test/testDelete.js

```diff
@@ -0,0 +1,125 @@
+process.env.NODE_ENV = 'test'
+const app = require('../server');
+let chai = require('chai');
+let chaiHttp = require('chai-http');
+chai.should();
+const fs = require('fs');
+chai.use(chaiHttp);
+
+const funcs = require('./testFunctions')
+const deleteAll = funcs.deleteAll;
+const addIssue = funcs.addIssue;
+const addIssueWithFile = funcs.addIssueWithFile;
+const addIssueWithFiles = funcs.addIssueWithFiles;
+
+
+describe('routes', function () {
+
+    beforeEach(() => {
+        deleteAll();
+    });
+
+    afterEach(() => {
+        deleteAll();
+    })
+
+
+
+    describe("DELETE /api/issues/delete/all", () => {
+        describe("One issue", () => {
+            it("status code 200, issue deleted", (done) => {
+                addIssue();
+                chai.request(app)
+                    .delete('/api/issues/delete/all')
+                    .end((err, res) => {
+                        res.should.have.status(200);
+                        res.body.should.be.an('object');
+                        res.body.should.have.property('ok').and.to.be.eql(1);
+                        res.body.should.have.property('n').and.to.be.eql(1);
+                        res.body.should.have.property('deletedCount').and.to.be.eql(1);
+                        done();
+                    })
+            })
```

Comment:
It would be nice to check if no issues exists after deletion is done

Reply:
Yes, I totally forgot about that.